R-packages/data-raw/create_rda.R

```diff
@@ -57,10 +59,10 @@ county_col_classes = c("numeric", "numeric", "character", "character",
                        "character", "character")
 county_file = system.file("extdata", "us_counties_centroids.csv", package = "usmap")
 county_geo = utils::read.csv(county_file, colClasses = county_col_classes)
-save(county_geo, file = "../covidcast/data/county_geo.rda", compress = "bzip2")
+#save(county_geo, file = "../covidcast/data/county_geo.rda", compress = "bzip2")
```

Comment:
Can you delete this line entirely?

Reply:
I wouldn't delete just this line. If we don't want this code saved anywhere, we should delete the whole block. I just did this part so it does not erronously create the output. Note: I did not write this file. Was just fixing an error that I found.